sbnanaobj/StandardRecord/StandardRecord.h

```diff
@@ -57,6 +58,8 @@ namespace caf
     std::vector<SRCRTSpacePoint> crt_spacepoints; ///< CRT spacepoints in event (SBND)
     int                         nsbnd_crt_tracks = 0; ///< Number of CRT tracks in event (SBND)
     std::vector<SRSBNDCRTTrack>  sbnd_crt_tracks; ///< CRT tracks in event (SBND)
+    SRSBNDCRTVeto                sbnd_crt_veto; ///< crt veto flags for each event
+    int                         nopflashes; ///< Number of OpFlashes in spill
```

Comment:
This looks like a merge conflict issue here? Duplication of lines. This worries me because it should have complained in the compiler? Has this been tested offline?